solr/core/src/test/org/apache/solr/handler/admin/ThreadDumpHandlerTest.java

```diff
@@ -245,21 +253,18 @@ public void doTestOwnableSync(final boolean checkWaitingThreadViaPolling) throws
     }
   }
 
-  @SuppressWarnings({"unchecked"})
-  private NamedList<Object> readProperties() throws Exception {
+  private void request(String path, Consumer<NamedList<?>> consumer) throws Exception {
     SolrClient client = new EmbeddedSolrServer(h.getCore());
-
-    NamedList<Object> properties = client.request(new GenericSolrRequest(SolrRequest.METHOD.GET, "/admin/info/threads",
-            new ModifiableSolrParams()));
-    return properties;
+    ModifiableSolrParams mparams = new ModifiableSolrParams();
+    mparams.set("indent", true);
+    NamedList<?> req = client.request(new GenericSolrRequest(SolrRequest.METHOD.GET, path, mparams));
```

Comment:
nit: would this be response, not request?

Reply:
yess, it should be response